cpp/posix.h

```diff
@@ -84,12 +84,12 @@ inline int close(int fd) {
 }
 
 inline int putenv(Str* name, Str* value) {
-  assert(0);
+  NotImplemented();
 }
 
 // TODO: errors
 inline int chdir(Str* path) {
-  assert(0);
+  NotImplemented(); // Uncalled
```

Comment:
Hm OK I think this was changed to pyos.Chdir.  So it's probably best to just delete these functions.

Reply:
"these functions" being all the ones in `posix.h` that are uncalled, or just `chdir` ?